repose-aggregator/core/core-lib/src/main/java/org/openrepose/core/services/rms/ImmutableFormatTemplates.java

```diff
@@ -43,8 +44,10 @@ private ImmutableFormatTemplates(List<StatusCodeMatcher> statusCodes) {
                 final String statusCodeId = statusCode.getId();
                 final String href = message.getHref();
                 final String stringTemplate = !StringUtilities.isBlank(href) ? new HrefFileReader().read(href, statusCodeId) : message.getValue();
-
-                formatTemplates.put(statusCodeId + message.getMediaType(), new HttpLogFormatter(stringTemplate));
+                final String mediaType = message.getMediaType();
```

Comment:
this should be content type not media type. media type has to do with what is coming back from the origin service. We care about what kind of message they are trying to build which is content-type